src/PKSim.Presentation/Presenters/Simulations/GlobalPKAnalysisPresenter.cs

```diff
@@ -164,6 +165,20 @@ public bool HasParameters()
          return false;
       }
 
+      public bool CanCalculateGlobalPK()
```

Comment:
I ran resharper to auto-format so there's some noise in this PR. The gist is we need to calculate when a simulation contains only multiple IV applications of all compounds. If there's only single IV, oral, or a mix we want to show the global panel.

Reply:
it this how it was doen for individual simulation?